### PR TITLE
Add logo repository

### PR DESCRIPTION
**Pull request by mark-cipolla**
Adds a page of easily downloadable logos, using the existing images in the repo.
It's not an exhaustive list of logos yet, but it gets the ball rolling.

window capture 1

### CODE REVIEW

**Comment by mark-cipolla** [comment]
[Add logo repository](https://trello.com/c/iAd2oW06/10902-add-logo-repository)

File: .storybook/FileDownload/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import styles from './styles.css';
+import file from './file.svg';
+
+class FileDownload extends Component {
```
Comment (nullobject) [inline .storybook/FileDownload/index.jsx]: Actually, there's no need for this to be a class. You could just use a function like we do for our other simple components.

**Comment by nullobject** [comment]
I realise that we're just adding stuff to this library at the moment, but how do we actually use it? How would I get hold of these logos if I use the tc-assets package in my app? Do they need to be referenced from an index file or something?

**Comment by mark-cipolla** [comment]
> I realise that we're just adding stuff to this library at the moment, but how do we actually use it? How would I get hold of these logos if I use the tc-assets package in my app? Do they need to be referenced from an index file or something?

Ah! Well, TC is already using the images from the tc-assets package.
I'm trying to build up a TC design style guide so non-tech staff can get logos from it.

They've got to be stored somewhere, and I'm tired of being asked for logos in various sizes; hence, trying to make a place where self-service is the go.

It's a long shot, and it'll need more work before it's comprehensive, but that's at least my thinking.

File: .storybook/FileDownload/index.jsx (hunk 1 above)
Comment (mark-cipolla) [reply .storybook/FileDownload/index.jsx]: I'll have a look (at how to structure this as a function) and restructure it.

**Comment by nullobject** [comment]
> Ah! Well, TC is already using the images from the tc-assets package.

Oh cool, I'll have a look at what TC is doing 👀

**Comment by nullobject** [comment]
Ah, I see. Because the TC `package.json` file is just requiring it from github, it includes _all_ the files. That makes sense.

If we ever published this as an NPM package then that wouldn't work, but that's not a problem right now.

File: .storybook/FileDownload/index.jsx (hunk 1 above)
Comment (mark-cipolla) [reply .storybook/FileDownload/index.jsx]: Done.

**Review by nullobject — APPROVED** [review]

**Pull request merged**